### ISSUE TITLE
How to perform trades?

### ISSUE THREAD

**Issue by gndelia** [BitGo/BitGoJS#3870]
I'm looking to trade some ETH to USDC (or at least, I'm first trying to do so in goerli testchain).

The REST API has some operations for that, but I can't find how to do so in the js SDK. Is it supported? Or should I go through the REST API?

**Comment by Antuan03** [comment]
you still need assistance ?

**Comment by gndelia** [comment]
@Antuan03 I was able to trade using the REST API, instead of the SDK  https://developers.bitgo.com/api/trade.orders.add I've got to admit it was a bit confusing, especially in the testing env, but I managed to make it work. Perhaps more working examples (both for the testing and prod envs) would be appreciated. Other than that, you can close this issue if you want

**Comment by Antuan03** [comment]
via the sdk 

`    const bitgo = new BitGo({
      accessToken: accessToken,
      env: 'test',
    });

    const wallet = await bitgo
      .coin(gtusdt)
      .wallets()
      .get({
        id: userWallet?.wallet_id,
      });

    let userParams = {
      amount: "amount in wei,
      address: 'address',
      walletPassphrase: userWallet?.passphrase,
    };

  const transaction1 = await wallet.send(userParams);
 
console.log( transaction1)
`

good luck!!

**Comment by gndelia** [comment]
@Antuan03 but that's to send crypto, not for trading, which is this issue about 🤔

**Comment by Antuan03** [comment]
oh my bad :x

**Comment by deepak2165** [comment]
I don't think we can perform trade using bitgo sdk. I have gone through the whole doc when i was looking for the same but didn't found any. So i performed the trade using API.
